frontera/contrib/backends/remote/messagebus.py

```diff
@@ -93,19 +97,65 @@ def states(self):
         return None
 
 
+class BufferedMixin(object):
+    def __init__(self, *args, **kwargs):
+        super_self = super(BufferedMixin, self)
+
+        super_self.__init__(*args, **kwargs)
+
+        self._buffer = OverusedBuffer(
+            super_self.get_next_requests,
+            self._logger.debug,
+        )
+
+    def get_next_requests(self, max_n_requests, **kwargs):
+        return self._buffer.get_next_requests(max_n_requests, **kwargs)
+
+
+class AggregatePerHostMixin(object):
```

Comment:
Is GenericMessageBusBackend usable without AggregatePerHostMixin? Please describe the use case.